pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -61,8 +61,6 @@ import (
 const (
 	controllerName = "mxjob-controller"
 
-	// mxJobCreatedReason is added in a mxjob when it is created.
```

Comment:
Just like to confirm these const are not being used right?

Reply:
According to the following golangci-lint logs, I think mxJobCreateReason is unused.

https://github.com/kubeflow/training-operator/runs/3829256127#step:9:17

> golangci-lint run --timeout 5m ./...
pkg/controller.v1/mxnet/mxjob_controller.go:65:2: `mxJobCreatedReason` is unused (deadcode)
	mxJobCreatedReason = "MXJobCreated"
	^